scenarios/evaluate-math/evaluate-math.ipynb

```diff
@@ -0,0 +1,274 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Evaluate with math evaluators\n",
+    "\n",
+    "## Objective\n",
+    "This notebook demonstrates how to use math-based evaluators to assess the quality of generated text by comparing it to reference text. By the end of this tutorial, you'll be able to:\n",
+    " - Understand different math evaluators such as `BleuScoreEvaluator`, `GleuScoreEvaluator`, `MeteorScoreEvaluator`, and `RougeScoreEvaluator`.\n",
+    " - Evaluate dataset using these evaluators.\n",
+    "\n",
+    "## Time\n",
+    "You should expect to spend about 10 minutes running this notebook.\n",
+    "\n",
+    "## Before you begin\n",
+    "\n",
+    "### Installation\n",
+    "Install the following packages required to execute this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install the packages\n",
+    "%pip install azure-ai-evaluation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set the following environment variables for use in this notebook:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os.environ[\"AZURE_SUBSCRIPTION_ID\"] = \"\"\n",
+    "os.environ[\"AZURE_RESOURCE_GROUP\"] = \"\"\n",
+    "os.environ[\"AZURE_PROJECT_NAME\"] = \"\""
+   ]
```

Comment:
Why not just configure the dictionary directly instead of using environment variables as a middle man?

```suggestion
    "\n",
    "azure_ai_project = {\n",
    "    \"subscription_id\": \"<YOUR-SUBSCRIPTION-ID>\",\n",
    "    \"resource_group_name\": \"<YOUR-AZURE_RESOURCE_GROUP>\",\n",
    "    \"project_name\": \"<YOUR-AZURE_PROJECT_NAME>\",\n",
    "}\n",
    "\n",
```

Reply:
Addressed.